icons.md

```diff
@@ -1,20 +1,21 @@
-|   Search   |  search   | <img src="https://user-images.githubusercontent.com/76241195/147102645-0d50ba65-5f21-4245-9275-6c3aec7d193c.png" width=100% height=30%>  |      Send      |      send      | <img src="https://user-images.githubusercontent.com/76241195/147096133-20e75d9a-0f96-4881-a165-5e7edf1bfe96.png"  width=100% height=30%> |
-|   Slack    |   slack   | <img src="https://user-images.githubusercontent.com/65664185/138503148-791f88ac-01ac-4d11-9a63-1ffaaf649b21.png" width=100% height=30%>  |    Snapchat    |    snapchat    | <img src="https://user-images.githubusercontent.com/91655303/148160774-755adc38-e089-4a20-910f-292b890e2c63.png" width=100% height=30%>  |
-|  Telegram  | telegram  | <img src="https://user-images.githubusercontent.com/65664185/138503468-8f27e3a9-d9ad-4348-85a4-d2c1761cd81f.png" width=100% height=30%>  |     TikTok     |     tiktok     | <img src="https://user-images.githubusercontent.com/76241195/147094674-a0b67f90-c62e-4162-93b5-dd313611ff28.png" width=100% height=30%>  |
-|   Twitch   |  twitch   | <img src="https://user-images.githubusercontent.com/76241195/147094354-12d3b38d-bb87-4c52-a3b7-4d934498da31.png" width=100% height=30%>  |    Twitter     |    twitter     | <img src="https://user-images.githubusercontent.com/65664185/138503209-1ce0ebbc-5590-4940-8cd0-2dadacbf09ed.png" width=100% height=30%>  |
-|   Vimeo    |   vimeo   | <img src="https://user-images.githubusercontent.com/65664185/138503257-6af44a9c-c81a-4657-b182-6a991157810f.png" width=100% height=30%>  |    YouTube     |    youtube     | <img src="https://user-images.githubusercontent.com/65664185/138503305-ff60cf54-6b0b-4e18-9446-b4f6869b9511.png" width=100% height=30%>  |
+|   Socials   |    Usage    | Icons                                                                                                                                                        |    Socials     |     Usage      | Icons                                                                                                                                   |
+| :---------: | :---------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------ | :------------: | :------------: | --------------------------------------------------------------------------------------------------------------------------------------- |
+|   Android   |   android   | <img src="https://user-images.githubusercontent.com/65664185/138502465-89cfadf2-6b54-4f3d-ac44-ceacdd4824ba.png" width=65% height=30%>                       |     Apple      |     apple      | <img src="https://user-images.githubusercontent.com/65664185/138502540-8e9b80bf-deae-4566-a41a-c63623e83c21.png" width=100% height=30%> |
```

Comment:
I think the `width` must be 100% here and for the other icons as well. Is that correct?
I still see a little bit of extra space in the icons.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Still the extra space does not get removed. If it is possible to remove it in any other way, anyone please edit the file.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

Reply:
> I think the `width` must be 100% here and for the other icons as well. Is that correct? I still see a little bit of extra space in the icons.
> 
> __ [![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966)](https://semasoftware.com/gh) [ ![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966) ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  ❓ I have a question

@Panquesito7 I think the space is okay for now. The changes are ready to be merged.__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Reusable, Maintainable